hack/lib/util/environment.sh

```diff
@@ -227,11 +227,17 @@ readonly -f os::util::environment::setup_etcd_vars
 #  - export SERVER_CONFIG_DIR
 #  - export MASTER_CONFIG_DIR
 #  - export NODE_CONFIG_DIR
+#  - export KUBE_CACHE_MUTATION_DETECTOR
+#  - export KUBE_PANIC_WATCH_DECODE_ERROR
 function os::util::environment::setup_server_vars() {
```

Comment:
All of our conformance/e2e testing runs off of a cluster that is set up by OpenShift-Ansible, not this script. So, you'll need to also add this var to the test job set-up.